### ISSUE TITLE
Update download page to offer 2.5

### ISSUE THREAD

**Issue by Totktonada** [tarantool/doc#1456]
Now [it](https://www.tarantool.io/en/download/os-installation/ubuntu/) contains 1.10 and 2.2. The latter is already EOL (2.2.3 is the last release of 2.2 series). Let's offer at least 2.4 (stable since 2.4.2) and 2.5 (beta since 2.5.1). Others are debatable (EOL and alpha).

All distro specific pages seems to be affected (and docker page too).

This is part of usual website maintainance activity, like #1450 and #1451.

**Comment by Totktonada** [comment]
It more or less related to #1337 (as @artembo pointed).

**Comment by Totktonada** [comment]
Now 2.5 reach the stable status: we released 2.5.2. Let's update instructions to offer the stable release branch and consider providing beta (2.6 now).

**Comment by Totktonada** [comment]
Fixed in the scope of #1641?

**Comment by artembo** [comment]
Done:

```
curl -L https://tarantool.io/installer.sh | bash

...

# tarantool --version
Tarantool 2.5.2-17-g67e52c1e9
```
